Fabulous.XamarinForms/src/Fabulous.XamarinForms/ViewConverters.fs

```diff
@@ -816,4 +816,11 @@ module Converters =
             target.ContentTemplate <- ViewElementDataTemplate(currValue)
             let realTarget = (target :> Xamarin.Forms.IShellContentController).Page
             if realTarget <> null then currValue.UpdateIncremental(prevValue, realTarget)            
-        | ValueSome _, ValueNone -> target.ContentTemplate <- null
\ No newline at end of file
+        | ValueSome _, ValueNone -> target.ContentTemplate <- null
+        
+    let updateUseSafeArea _ (currValue: bool voption) (target: Xamarin.Forms.Page) =
+        match currValue with
```

Comment:
the previous Value should be used, too. You can check the other updateCodes on the behavior.

this is just so fabulous won't call functions if they don't need to be called. Srsly I am not good at explaining .....

Reply:
Don't beat yourself up 😄 
It was understandable, at least for me. :)

To be efficient, Fabulous.XamarinForms uses some view diffing techniques to avoid calling Xamarin.Forms for nothing.
And comparing the previous value vs the current one help call Xamarin.Forms only when the value really changed.
That's why you can see other functions following the same pattern of matching both prevValue and currValue.

Reply:
It made sense to me! I'll make the change.